gviz_api.py

```diff
@@ -495,9 +500,9 @@ def TableDescriptionParser(table_description, depth=0):
     # dictionary).
     # NOTE: this way of differentiating might create ambiguity. See docs.
     if (len(table_description) != 1 or
-        (isinstance(table_description.keys()[0], types.StringTypes) and
-         isinstance(table_description.values()[0], tuple) and
-         len(table_description.values()[0]) < 4)):
+        (isinstance(sorted(table_description.keys())[0], six.string_types) and
+         isinstance(sorted(table_description.values())[0], tuple) and
+         len(sorted(table_description.values())[0]) < 4)):
```

Comment:
Consider using next(iter(table_description)) to get some key and next(six.itervalues(table_description)) to get some value instead of actually sorting (which duplicates the data and sorts it). You can also use six.iterkeys if you prefer it to be more consistent.

Same comment below.